styles/header.css

```diff
@@ -0,0 +1,290 @@
+  flex-direction: column;
+  align-items: center;
+  padding: 0.75rem;
+}
+
+.menu-btn {
+  position: absolute;
+  z-index: 22;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  justify-content: center;
+  align-self: flex-start;
+  background-color: transparent;
+  height: 40px;
+  transition: 0.5s ease-in-out;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 35px;
+  height: 4px;
+  background-color: var(--color-black);
+  border-radius: 5px;
+  transition: all 0.5s ease-in-out;
+}
+
+.top-navigation {
+  gap: 2rem;
+  display: none;
+}
+
+.top-navigation li {
+  align-items: flex-start;
+  list-style: none;
+}
+
+.top-navigation a {
+  color: var(--color-white);
+}
+
+.nav-bar a {
+  color: var(--color-black);
+}
+
+.top-navigation a:hover {
+  color: var(--color-red);
+}
+
+.top-navigation svg {
+  width: 1.5rem;
+  height: 1.5rem;
+  transition: transform 0.275s ease-in-out;
+}
+
+.top-navigation svg:hover {
+  transform: translateY(-8px) scale(1.2);
+}
+
+.top-navigation svg:hover path {
+  fill: var(--color-red);
+}
+
+.bottom-navigation {
+  display: none;
+  background-color: var(--color-white);
+  align-items: center;
+  padding: 0 4rem;
+}
+
+.bottom-navigation img {
+  width: 130px;
+  object-fit: cover;
+}
+
+.nav-bar {
+  display: inline-flex;
+  gap: 2rem;
+}
+
+.nav-bar span {
+  border: 5px solid var(--color-red);
+  padding: 10px;
+  color: var(--color-red);
+}
+
+.active-link {
+  color: var(--color-red) !important;
+}
+
+/* Mobile Menu Section */
+
+.mobile-menu {
+  position: fixed;
+  right: -200%;
+  top: 0;
+  z-index: 22;
+  flex-direction: column;
+  background-image: url("../images/Background/conference-background-1.jpg");
+  height: 100vh;
+  width: 100%;
+  text-align: start;
+  padding: 4rem;
+  transition: 0.95s;
+  filter: drop-shadow(20px);
+  opacity: 0.82;
+}
+
+.mobile-menu.active {
+  right: 0;
+}
+
+.mobile-menu .nav-bar {
+  display: flex;
+  flex-direction: column;
+}
+
+.mobile-menu .nav-bar a {
+  font-weight: 600;
+  font-size: 2rem;
+  line-height: 2.75rem;
+  color: var(--color-black);
+  opacity: 0.8;
+}
+
+.mobile-menu .nav-bar a:first-child {
+  display: flex;
+  align-items: flex-end;
+  opacity: unset;
+}
+
+.menu-btn.open .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.mobile-menu .nav-bar span {
+  color: var(--color-black);
+  padding: unset;
+  border: unset;
+}
+
+/* Header Section */
+
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 0.75rem;
+  background-image: url("../images/Background/conference-background-2.webp");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  width: 100vw;
+  padding: 80px 0.9375rem;
+}
+
+.slogan {
+  font-size: 1.5rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+  color: var(--color-red);
+}
+
+.title {
+  font-family: var(--ff-cocogoose);
+  color: var(--color-red);
+  font-size: 2.5rem;
+  font-weight: 700;
+  line-height: 3.25rem;
+  width: max-content;
```

Comment:
- [OPTIONAL] On the 'Home' page mobile view, the title is larger than some screen sizes making a horizontal scroll possible. This generates a bad user experience and makes the page look not professional. This might be caused by the large title. I would change that to make the page fits in every mobile screen.
 - Your mobile view:
<img width="484" alt="image" src="https://user-images.githubusercontent.com/61919617/190455304-c8b0cdf9-df46-4f21-a6dd-7ba8dd35dba4.png">